Dockerfiles/agent/Dockerfile

```diff
@@ -10,12 +10,16 @@ WORKDIR /output
 #   - static libraries                   # FIXME: move upstream
 #   - jmxfetch on nojmx build
 RUN dpkg -x /datadog-agent*_amd64.deb . \
+ && echo "Cleanup and trimming" \
  && rm -rf usr etc/init lib \
     opt/datadog-agent/sources \
     opt/datadog-agent/embedded/share/doc \
     opt/datadog-agent/embedded/share/man \
  && find opt/datadog-agent/ -iname "*.a" -delete \
- && if [ -z "$WITH_JMX" ]; then rm -rf opt/datadog-agent/bin/agent/dist/jmx; fi
+ && if [ -z "$WITH_JMX" ]; then rm -rf opt/datadog-agent/bin/agent/dist/jmx; fi \
+ && echo "Linking embedded ssl certificates" \
+ && mkdir -p etc \
```

Comment:
Not sure about the mkdir etc ? (and the `-p` option could be removed)

Reply:
`etc` should indeed be in the package, but if it's not the `ln` invocation will fail. The `-p` makes `mkdir` ignore the error if it already exists. As `mkdir` has no other "ignore errors" flag, the alternative is to write a shell if condition.

Reply:
Don't we want it to fail if we have no `etc`? 🤔

Reply:
you have a point, `/etc/datadog-agent` should be present anyway ;)